perception/sandbox/polygrasp/src/polygrasp/robot_interface.py

```diff
@@ -0,0 +1,151 @@
+"""polymetis.RobotInterface combined with GripperInterface, with an additional `grasp` method."""
+
+import time
+import numpy as np
+import logging
+from scipy.spatial.transform import Rotation as R
+import torch
+
+import hydra
+import graspnetAPI
+import polymetis
+
+log = logging.getLogger(__name__)
+
+
+def compute_des_pose(best_grasp):
+    grasp_point = best_grasp.translation
+
+    grasp_approach_delta = best_grasp.rotation_matrix @ np.array([-0.3, 0.0, 0])
+    grasp_approach_delta_plus = best_grasp.rotation_matrix @ np.array([-0.3, 0.1, 0])
+    grasp_approach_delta_minus = best_grasp.rotation_matrix @ np.array([-0.3, -0.1, 0])
```

Comment:
What's with these vectors? Should they be defined in the configs so users can easily tune them?